sklearn/tree/_classes.py

```diff
@@ -381,7 +381,7 @@ def fit(self, X, y, sample_weight=None, check_input=True,
         return self
 
     def _validate_X_predict(self, X, check_input):
-        """Validate X whenever one tries to predict, apply, predict_proba"""
+        """Validate X whenever one tries to predict, apply, predict_proba."""
```

Comment:
The short summary should be a one-line summary that does not use variable names or the function names.

Maybe we could use:

```suggestion
        """Validate the training data on predict (probabilities)."""
```